### ISSUE TITLE
Screen reader is not announcing the 'Equations' in solution information while navigating in scan mode.

### ISSUE THREAD

**Issue by msftedad** [mathjax/MathJax-src#1287]
**Test Environment:**
Window 11 24H2 26100.2605
Edge Browser Version 137.0.3296.93 
Tools used: NVDA, Narrator, JAWS
Test URL: https://codepen.io/trcko/pen/YzpevPm

**User Impact:**
When navigating in scan mode, the screen reader's silence on variables and special characters can significantly affect users who rely on it for information. This lack of auditory feedback can make it difficult for them to understand the content and context of the page, leading to a frustrating and inaccessible experience. Ensuring that all elements are announced is crucial for an inclusive user experience.

**Observation:** 
Issue is observed with all screen readers i.e., Narrator, NVDA and JAWS.
While navigating in browse mode using nvda screen reader is remaining silent when focus is on the variables and special characters in the equation.

**Repro Steps:**

1. Open the URL - https://codepen.io/trcko/pen/YzpevPm on edge browser.
2. Turn on Narrator using ‘Ctrl + Win + Enter’ keys.
3. Press 'Caps lock + spacebar’ key to navigate in scan mode. 
4. Use arrow keys to navigate over the mathematical expression. 
5.  Observe the screen reader announcement and verify whether Screen reader is announcing the 'Equations' in solution information while navigating in scan mode.

**Actual Result:**
While navigating in scan mode screen reader is remaining silent i.e., not announcing any information when focus is on the variables and special characters in the page. For example, when navigating over expressions , the screen reader fails to announce the variable 'x', ‘y’. 

**Refer Attachment:** 
1.While navigating in scan mode screen reader is remaining silent i.e., not announcing any information when focus is on the variables and special characters in the page..png
2.In scan mode, when users navigate over mathematical expressions, the screen reader remains silent.mp4

https://github.com/user-attachments/assets/8da15dda-07a2-4ecc-a470-dd20b744ee55
<img width="960" alt="Image" src="https://github.com/user-attachments/assets/230266ed-f61b-493c-9604-56920d76ba24" />

**Expected result:**
While navigating in scan mode screen reader should be announcing the variables and special characters information in the equation when focus is on the content in the page. 
The screen reader should announce each component of the equation, including variables like 'x,' providing users with accurate mathematical expressions. 
Mathematical symbols and operations should be pronounced correctly to ensure comprehension.

**Comment by dpvc** [comment]
By default, MathJax v3 inserts MathML into the page that is visually hidden but available to screen readers.  It looks like the Windows screen readers are not accessing this.

The alternative is to activate the MathJax accessibility tools using the Accessibility submenu of the MathJax contextual menu.  That will cause MathJax to produce its own speech text for the screen readers, and that seems to work for me in your example (at least for NVDA; I did not try the others).

MathJax v4 (now out in beta release) has the accessibility tools on by default.  We are preparing a new release that should be out this week that includes major updates to the assistive tools, and they should work more smoothly with screen readers.

**Comment by dpvc** [comment]
I'm moving this to the MathJax issue tracker, since the issue is about using mathjax, not the structure of this source directory.
